eZ/Publish/API/Repository/Tests/SearchService/DeleteTranslationTest.php

```diff
@@ -123,4 +125,38 @@ public function testDeleteContentTranslation(): void
         $searchResult = $this->findContent('Kontakt', 'ger-DE');
         $this->assertEquals(0, $searchResult->totalCount);
     }
+
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testPreventTranslationDeletionIfNoAccess(): void
```

Comment:
It would be nice to have a test which will check if a user with proper permission can delete translation (not an administrator)

Reply:
@mikadamczyk  resolved in https://github.com/ezsystems/ezpublish-kernel/pull/3036/commits/4064f062c6fc40d651b2783c4e1d5a58fa2b4f00